README.md

```diff
@@ -40,10 +40,9 @@ Authorization](https://cloud.google.com/binary-authorization/).  For details see
 
 * Watch the talk on [Software Supply Chain Management with Grafeas and Kritis](https://www.infoq.com/presentations/supply-grafeas-kritis/)
 * Learn the concepts in the [Kritis whitepaper](docs/binary-authorization.md)
-* Get Kritis running with the [Installation guide](docs/install.md)
-* Try the [Tutorial](docs/tutorial.md) to learn how to block vulnerabilities
+* Try out Kritis with standalone Grafeas by following [instructions](docs/standalone/README.md)
+* Try the [Tutorial](docs/tutorial.md) to learn how to block vulnerabilities using [GCP Container Analysis](https://cloud.google.com/container-registry/docs/get-image-vulnerabilities)
```

Comment:
[minor] Similarly for [tutorial]. Also maybe we should just stick to either instruction or tutorial, but not both.